### ISSUE TITLE
Add ES7 Examples

### ISSUE THREAD

**Issue by joshghent** [esfiddle/esfiddle#13]
We currently have a list of ES6 example code featuring the new functionality of ES6.

Now that ES7's spec is out then we need to add an example for each feature that has been released.

You can find the new ES7 features below:
http://www.2ality.com/2016/02/exponentiation-operator.html
http://www.2ality.com/2016/02/array-prototype-includes.html

The examples dropdown will need an "ES7 Examples" label divider and then an option for each example.

**Comment by ageddesi** [comment]
Is this still up for grabs?

**Comment by joshghent** [comment]
@ageddesi Sure is! We'd happily welcome PR's 👍

**Comment by ageddesi** [comment]
I was doing good but got blocked by mongodb. I have not used this before so
may be a bit while I work out how to set this up and change the default URL
as outlined in your steps.

**Comment by joshghent** [comment]
@ageddesi That's ok, why do you need to change the default URL?

**Comment by ageddesi** [comment]
Sorry for not being clearer, I have never used mongdb before so I am at lost what to do here. I have got mongodb installed on my mac and running . 

Configure mongo in your environment:
MONGOHQURL='mongodb://localhost:27017/fiddles' or something similar

**Comment by ageddesi** [comment]
I have done the fix on my fork, https://github.com/ageddesi/esfiddle, however, I just noticed that the files in the project all seemed to be changed by my editor :( Fail!

Will try again tomorrow on a fresh branch and turn off my editors' file formatting.

**Comment by joshghent** [comment]
@ageddesi Ah yes that's a pain. In the future we will be distributing an .editorconfig with the project to stop this stuff happening! Thanks for your work, look forward to your PR tomorrow.

**Comment by ageddesi** [comment]
This has now been merged into the master branch. Suggest closing this ticket.

**Comment by joshghent** [comment]
Yes forgot to close this.

This has been done in https://github.com/esfiddle/esfiddle/commit/4f41333ff403fbf682c0a86ba77c1df83783d151
